src/components/interfaces/MOBILE_API.xml

```diff
@@ -1847,55 +1847,55 @@
         </param>
         <param name="latitudeDegrees" type="Float" minvalue="-90" maxvalue="90" mandatory="true">
         </param>
-        <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="true">
+        <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="false">
```

Comment:
`history` tag must be added.

Reply:
Please see https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0089-mobile-api-versioning.md#scenario-3-change-signature-of-sub-element-param

Reply:
This must be done for each changed parameter, here is the appropriate changes for this specific parameter:
```
        <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="false" since="5.0">
            <description>The current UTC year.</description>
            <history>
                <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="true" since="2.0" until="5.0">
            </history>
        </param>
```